docs/html-css/week-3/learning-objectives.md

```diff
@@ -6,40 +6,43 @@ sidebar_label: Learning Objectives
 
 ## Forms
 
-### What are forms?
-
 - [ ] Can answer: what are HTML forms used for?
 
-### The HTML Form
+### Form controls
+
+- [ ] Can answer: what form controls can we use when we want the user to enter text
+- [ ] Can answer: what form controls can we use when we want the user to choose a single option from a list
+- [ ] Can answer: what form controls can we use when we want the user to choose multiple options from a list
+- [ ] Can: identify the type of form input likely in use in a sample form
+- [ ] Can: select the correct form inputs when building a simple form
 
-- Fields
-- Labels
-- Buttons
+### Labels
 
-- [ ] Can answer: what happens when the user clicks on a form's Submit button?
+- [ ] Can answer: why are labels necessary?
+- [ ] Can answer: in which ways do form control labels benefit users?
+- [ ] Can answer: how do we associate a label with a form control?
+- [ ] Can answer: how do we give a group of radio or checkbox inputs a label
+- [ ] Can answer: why are text input placeholders bad?
 
-- More Field Types
+### Form validation
 
-- [ ] Can: identify the type of form input likely in use in a sample form.
-- [ ] Can: select the correct form inputs when building a simple form.
+- [ ] Can answer: what are some attributes we can use for form validation?
 
-- Fieldsets
-- Form field Attributes
+### Form submission
 
-- [ ] Can answer: What's the purpose of a form field's `required` attribute? What about its `readonly`, `value`, or `checked` attributes?
+- [ ] Can answer: in which ways can we submit a form?
+- [ ] Can answer: how can we submit a form to a web server?
+  
+### Form security
 
-- [ ] Can answer: When should you use GET or POST as a form's `method` attribute?
+- [ ] Can answer: what should we never trust in software?
```

Comment:
This is very open - what answer are we expecting here?

Reply:
The form security section states that we should never trust user input. Although it's touched upon briefly, it's very important for all developers to know this

https://github.com/CodeYourFuture/syllabus/blob/master/docs/html-css/week-3/lesson.md#form-security:

> In software, we follow the rule that we **never** trust user input